pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/IndexLoadingConfig.java

```diff
@@ -49,6 +49,7 @@
 public class IndexLoadingConfig {
   private static final int DEFAULT_REALTIME_AVG_MULTI_VALUE_COUNT = 2;
   public static final String DEFAULT_TIER_BACKEND = "local";
+  private static final int DEFAULT_RANGE_INDEX_VERSION = 1;
```

Comment:
+1 on putting one as the default version. Once all pinot components are deployed, user can start generating the new index version

Reply:
This has moved to `IndexingConfig`